team-builders/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+// import logo from './logo.svg';
+import './App.css';
+import FormInput from "./components/FormInput";
+import FormCard from "./components/FormCard";
+
+function App() {
+  const [members, setMembers] = useState([
+    {
+      id: "",
+      role: "",
+      email: ""
+    }
+  ]);
+
+  const addNewMember = member => {
+    const newMembers = {
+      id: Date.now(),
+      name: member.name,
+      email: member.email,
+      role: member.role
+    };
+
+    setMembers([...members, newMembers]);
```

Comment:
Pop Quiz: How would you set it up so that the form wouldn't allow someone to submit an empty card?